autotest/OldTests/src/test/java/com/tle/webtests/test/contribute/ItemUnlock.java

```diff
@@ -9,16 +9,15 @@
 import com.tle.webtests.pageobject.wizard.ContributePage;
 import com.tle.webtests.pageobject.wizard.WizardPageTab;
 import com.tle.webtests.pageobject.wizard.controls.UniversalControl;
-import com.tle.webtests.pageobject.wizard.controls.universal.YouTubeUniversalControlType;
+import com.tle.webtests.pageobject.wizard.controls.universal.UrlUniversalControlType;
 import com.tle.webtests.test.AbstractCleanupTest;
 import org.testng.annotations.Test;
 
 @TestInstitution("fiveo")
 public class ItemUnlock extends AbstractCleanupTest {
-  private static final String RENAMED_NAME = "A Video";
-  private static final String ORIGINAL_NAME =
-      "What is a function? | Functions and their graphs | Algebra II | Khan Academy";
-  private static final String DISPLAY_NAME = "Original Displayname";
+  private static final String RENAMED_NAME = "AMAZON";
```

Comment:
Nit: Maybe just go with something less outlandish like

```suggestion
  private static final String RENAMED_NAME = "Google Renamed";
```